### PR TITLE
Add info about pipeline resources

### PR DESCRIPTION
**Pull request by nicoloboschi**

### CODE REVIEW

File: building-applications/pipelines.md [hunk 1]
```diff
@@ -27,10 +25,87 @@ pipeline:
...
+Setting `parallelism:` to 3 the pipeline will be deployed with 3 different replicas. Each replica will use the configured `size` for CPU and memory.
 
-### Pipeline Configuration Values
+The `size` parameter comprehend both CPU and memory. The value is a multiplier factor that is computed at runtime, starting from a base cpu/memory value.
+For example, if the default value for the memory - for a single replica - is 512 MB, specifying `size:  2` will make the pipeline to use 1024MB per-replica.
```
Comment (eolivelli) [inline building-applications/pipelines.md:108]: where do I configure the values ?

can we add a section about it ?
we can write it here, or in a separate file and link it.

Comment (mendonk) [reply building-applications/pipelines.md:108]:
```suggestion
For example, if the default value for the memory - for a single replica - is 512 MB, specifying `size: 2` will make the pipeline use 1024MB per-replica, as below:
```yaml
name: "Write to Cassandra"
topics:
    - name: "input-topic"
      creation-mode: create-if-not-exists
pipeline:
  - name: "Write to Cassandra"
    type: "sink"
    input: "input-topic"
    resources:
      size: 2
```
The default value for `parallelism` and `resources` is `1`. 
```

**Review by mendonk — APPROVED** [review]
+1

**Comment by nicoloboschi** [comment]
@eolivelli @mendonk the default value are configurable in the "admin" side. We don't have the admin guide yet. There's no sight of how to configure the deployer so I think it's better to not include that in this page for now. Maybe we can add a link once we have the admin config

**Pull request merged**